core/pv-mqtt/src/main/java/org/phoebus/pv/mqtt/MQTT_PV.java

```diff
@@ -205,9 +205,8 @@ public void messageArrived(String topic, MqttMessage msg) throws Exception
         }
         catch (Exception ex)
         {
-            logger.log(Level.SEVERE, "Could not parse message: '" + new_value + "' to " + getName());
-
-            throw new Exception("Failed to parse message", ex);
+            notifyListenersOfDisconnect();
+            logger.log(Level.WARNING, "Could not parse message: '" + new_value + "' to " + getName());
```

Comment:
Only logging the issue, not throwing an exception, may be the better approach here.
But since the original `Exception ex` is no longer wrapped and passed up, add it to the log so we see the underlying cause in the log:
```
logger.log(Level.WARNING, "Could not parse message: '" + new_value + "' to " + getName(), ex);
```